listener/upload_listener.py

```diff
@@ -380,17 +388,23 @@ def process_message(msg):
             return
         FailedCache.process_failed_cache(FailedCache.upload_cache, executor, process_upload, loop)
         FailedCache.process_failed_cache(FailedCache.delete_cache, executor, process_delete, loop)
+        # send message to payload tracker
+        send_msg_to_payload_tracker(PAYLOAD_TRACKER_PRODUCER, msg_dict, 'received', loop=loop)
```

Comment:
it sends for every message received, not only uploaded payloads, it may miss some required params in msg_dict, move it under `if msg.topic == mqueue.UPLOAD_TOPIC:` at least